index.html

```diff
@@ -0,0 +1,645 @@
+    </nav> -->
+
+<nav class="mobile-nav">
+    <img src="assets/images/logo.png" alt="">
+    <section class="mainNav-right">
+         <i class="fas fa-search"></i>
+         <i class="fas fa-bars"></i>
+        </section>
+</nav>
+<nav class="top-nav">
+        <section class="topNav-left">
+            <a href="#">New</a>
+            <a href="#">Events</a>
+            <a href="#">Business</a>
+            <a href="#">AMAs</a>
+            <a href="#">Space</a>
+            <a href="#">Terms & Conditions</a>
+        </section>
+        <section class="topNav-right">
+            <a href="#">About</a>
+            <a href="#">Advertise</a>
+            <a href="#">Jobs</a>
+            <a href="#">Conact</a>
+        </section>
+    </nav>
+        <nav class="main-nav">
+            
+        <section class="mainNav-left">
+            <img class="nav-logo" src="assets/images/logo.png" alt="">
+            <a href="#">Latest</a>
+            <a href="#">Hard Work</a>
+            <a href="#">Pluged</a>
+            <a href="#">Readme</a>
+            <a href="#">Growth Quarters</a>
+            <a href="#">Shift</a>
+            <a href="#">Neural</a>
+        </section>
+        <section class="mainNav-right">
+         <i class="fas fa-search"></i>
+        </section>
+    </nav>
+    
+
+    <main>
+
+<section class="header-img">
+    <div class="main-header-img" style="background-image: url(assets/images/40.png);">
+    <div class="logo-date">
+        <div class="cover-logo" style="background-image: url(http://www.w3.org/2000/svg);"></div>
+        <time class="article-date">Webnesday-January 20th 2021</time>
+    </div>
+                <div class="cover-text">                
+                    <h3 class="cover-title">GitHub offers to rehire employee it fired for calling insurrectionists ‘Nazis’</h3>
+                    <div class="byline">
+                        <small class="article-small">by Thomas Macaulay </small>
+                        <small class="article-small"><i class="far fa-clock"></i>1 hour ago </small>
+                        <small class="article-small"><i class="far fa-clock"></i> 4</small>
+                    </div>
+                </div>
+             <div class="img-overlay"></div>
+    </div>
+    <div class="sub-header-img">
+        <div style="background-image: url(assets/images/4.png);">
+                <div class="sub-img-text">
+                <time>TECH</time>
+                <h3>Astronomers just detected the oldest and most distant quasar ever</h3>
+                    <div>
+                        <small>by Thomas Macaulay </small>                
+                        <small><i class="far fa-clock"></i>3 hour ago </small>
+                        <small><i class="far fa-clock"></i>12</small>
+                    </div>
+                <div class="img-overlay"></div>
+            </div>
+        </div>
+        <div style="background-image: url(assets/images/4.png);"> 
```

Comment:
## No inline styling

Using Inline Styles is probably one of the worst CSS practices that many web designers and developers are still doing to this day. It transgresses the main goal of why we use CSS – which is to separate style from the HTML structure. This breaks the specificity of web pages

- [ ] Remove all `style` inline you have in the HTML file
- [ ] Use an id as identifier and add in the css file
```html
<div id="third-article">
```
```css
  #third-article{
     background-image('src');
}
```
**Note:** Remove all inline style rules in the entire HTML page


###### _[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_